packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.jsx

```diff
@@ -1,55 +1,127 @@
+                    'contract-type-info__card--has-toggle-buttons': !has_toggle_buttons,
+                })}
+            >
                 <ThemedScrollbars
-                    className='contract-type-info__scrollbars'
-                    height={isMobile() ? '' : '300px'}
+                    className={classNames('contract-type-info__scrollbars', {
+                        'contract-type-info__scrollbars-description--active': is_description_tab_selected,
+                        'contract-type-info__scrollbars-glossary contract-type-info__scrollbars-glossary--active ':
+                            is_glossary_tab_selected,
+                    })}
+                    style={{
+                        left: `${is_description_tab_selected ? `-${width}` : width}px`,
+                        transform: `translate3d(${is_description_tab_selected ? width : `-${width}`}px, 0, 0)`,
+                    }}
+                    height={isMobile() ? '' : scroll_bar_height}
                     autohide={false}
                 >
+                    <div
+                        className={classNames({
+                            'contract-type-info__gif--has-toggle-buttons': has_toggle_buttons,
+                            'contract-type-info__content': is_glossary_tab_selected,
+                            'contract-type-info__gif': is_description_tab_selected,
+                        })}
+                    >
+                        {is_description_tab_selected ? (
+                            <TradeCategoriesGIF category={type.value} />
+                        ) : (
+                            <ContractTypeGlossary category={type.value} />
+                        )}
+                    </div>
                     <div className='contract-type-info__content'>
-                        <TradeCategories category={type.value} />
+                        {is_description_tab_selected && (
+                            <TradeCategories category={type.value} onClick={onClickGlossary} />
+                        )}
                     </div>
                 </ThemedScrollbars>
+            </div>
+        );
+    });
+
+    return (
+        <React.Fragment>
+            {!has_toggle_buttons && (
+                <div className='contract-type-info__button-wrapper'>
+                    <ButtonToggle
+                        buttons_arr={[
+                            { text: localize('Description'), value: TABS.DESCRIPTION },
+                            { text: localize('Glossary'), value: TABS.GLOSSARY },
+                        ]}
+                        name='description_glossary_filter'
+                        is_animated
+                        has_rounded_button
+                        onChange={e => {
+                            setSelectedTab(e.target.value);
+                        }}
+                        value={selected_tab}
+                    />
+                </div>
+            )}
+            <Carousel
```

Comment:
I'm still not sure if <Carousel /> should remain here. The task was to remove it, not just to hide.
Anyway disabling and hiding <Carousel /> seems even not semantic + might cause misunderstandings.

Reply:
Because the requirement was to hide the navigation and pagination components. At least thats how i understood and agreed upon.